File: TestingIntegrationBetweenObjectiveCAndSwift/TestingIntegrationBetweenObjectiveCAndSwift/ViewController.m
//
//  ViewController.m
//  TestingIntegrationBetweenObjectiveCAndSwift
//
//  Created by Thomaz Avila Barros on 01/05/16.
//  Copyright © 2016 ThomazBarros. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

- (IBAction) someMethod {
    MySwiftObject * myOb = [MySwiftObject new];
    NSLog(@"MyOb.someProperty: %@", myOb.someProperty);
    myOb.someProperty = @"Hello World";
    NSLog(@"MyOb.someProperty: %@", myOb.someProperty);
    NSString * retString = [myOb someFunction:@"Arg"];
    NSLog(@"RetString: %@", retString);
}

@end

File: Dog Age Calculator/Dog Age Calculator/ViewController.swift
//
//  ViewController.swift
//  Dog Age Calculator
//
//  Created by Thomaz Avila Barros on 18/09/16.
//  Copyright © 2016 ThomazBarros. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var ageTextField: UITextField!
    @IBOutlet weak var resultLabel: UILabel!
    @IBAction func discoverAgeButton(_ sender: AnyObject) {
        print("Button clicked.")
        let age = Int(ageTextField.text!)!
        resultLabel.text = "Your dog age is "+String(age*7)+" years old."
        
    }
    
    
    override func viewDidLoad() {
        super.viewDidLoad()

        print("The app has been loaded.")

    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}

